### PR TITLE
ethereum liquidate

### PR DESCRIPTION
**Pull request by aniiantt**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
     {
-      "name": "Current TS Tests File",
       "type": "node",
       "request": "launch",
-      "args": ["-r", "ts-node/register", "${relativeFile}"],
+      "name": "Launch Program",
       "cwd": "${workspaceRoot}",
-      "protocol": "inspector"
+      "runtimeArgs": ["-r", "ts-node/register"],
+      "args": ["${workspaceRoot}/examples/liquidation_bot/index.ts"]
```
Comment (xlc) [inline .vscode/launch.json:13]: have both config?

File: examples/liquidation_bot/config.yml [hunk 2]
```diff
@@ -0,0 +1,167 @@
+version: '0.1'
+guardians:
+  ethereum:
+    networkType: ethereum
+    network: kovan
+    nodeEndpoint: wss://kovan.infura.io/ws/v3/16a5aa3a08c24d56b1586cd06b4055d5
+    monitors:
+      syntheticPoolGeneral-FEUR:
+        task: synthetic.pools
+        arguments:
```
Comment (xlc) [inline examples/liquidation_bot/config.yml:10]: can you make one task monitor all pools?

Comment (aniiantt) [reply examples/liquidation_bot/config.yml:10]: > can you make one task monitor all pools?
Does it also include all tokens?

Comment (xlc) [reply examples/liquidation_bot/config.yml:10]: few changes:

- make the token name can be an array, so user can pass `[FEUR, FJPY]`
- the SDK should include all token names
- add a alias so `all` translates to `[FEUR, FJPY, ...]` all tokens

**Pull request merged**